assets/scripts/oppia-player-0.0.0.js

```diff
@@ -244,9 +244,8 @@
     var versionString = explorationVersion ? '&v=' + explorationVersion : '';
     iframe.setAttribute('src', encodeURI(
       (oppiaNode.getAttribute('src') || currLoc) +
-      '/explore/' + oppiaNode.getAttribute('oppia-id') +
-      '?iframed=true&locale=en' + versionString +
-      '#' + TAG_ID_KEY + tagId +
+      '/embed/exploration/' + oppiaNode.getAttribute('oppia-id') +
```

Comment:
Ditto for anything in assets/scripts -- that can't be changed. We need to make a new version of the embed script and update our instructions accordingly. We also need to redirect ...?iframed=true to the new embed URL.